api_formatter/templates/rest_framework/api.html

```diff
@@ -1,292 +1,415 @@
-        <span class="navbar-toggler-icon"></span>
-      </button>
+    <nav class="nav-right" aria-label="Main">
+      <ul class="nav__list show-on-lg-up">
+        <li class="nav__item btn--navy">
+          <a class="nav__link" href="{% url 'agent-list' %}" id="agents">
+            Agents
+          </a>
+        </li>
+        <li class="nav__item btn--navy">
+          <a class="nav__link" href="{% url 'collection-list' %}" id="collections">
+            Collections
+          </a>
+        </li>
+        <li class="nav__item btn--navy">
+          <a class="nav__link" href="{% url 'object-list' %}" id="objects">
+            Objects
+          </a>
+        </li>
+        <li class="nav__item btn--navy">
+          <a class="nav__link" href="https://docs.rockarch.org/argo-docs" id="docs">
+            Documentation
+            <span class="material-icon" aria-hidden="true">launch</span>
+          </a>
+        </li>
+      </ul>
 
-      <div class="collapse navbar-collapse" id="navbarSupportedContent">
-        <ul class="navbar-nav mr-auto">
-          <li class="nav-item"><a class="nav-link text-light" href="{% url 'agent-list' %}">Agents</a></li>
-          <li class="nav-item"><a class="nav-link text-light" href="{% url 'collection-list' %}">Collections</a></li>
-          <li class="nav-item"><a class="nav-link text-light" href="{% url 'object-list' %}">Objects</a></li>
-          <li class="nav-item"><a class="nav-link text-light" href="https://docs.rockarch.org/argo-docs">Documentation</a></li>
+      <div class="dropdown hide-on-lg-up">
+        <button id="nav-toggle" class="btn btn--navy nav__btn--mobile closed" tabindex="0" aria-expanded="false">
+          <span class="material-icon">menu</span>
+        </button>
+        <ul id="nav-toggle-menu" class="dropdown__list dropdown__list--mobile list--unstyled dropdown__list--navy dropdown__list--slide-left closed">
+          <li>
+            <a id="menu-item-agents" class="btn btn--navy dropdown__btn dropdown__btn--mobile" href="{% url 'agent-list' %}" tabindex="-1">
+              Agents
+            </a>
+          </li>
+          <li>
+            <a id="menu-item-collections" class="btn btn--navy dropdown__btn dropdown__btn--mobile" href="{% url 'collection-list' %}" tabindex="-1">
+              Collections
+            </a>
+          </li>
+          <li>
+            <a id="menu-item-objects" class="btn btn--navy dropdown__btn dropdown__btn--mobile" href="{% url 'object-list' %}" tabindex="-1">
+              Objects
+            </a>
+          </li>
+          <li>
+            <a id="menu-item-documentation" class="btn btn--navy dropdown__btn dropdown__btn--mobile" href="https://docs.rockarch.org/argo-docs" tabindex="-1">
+              Documentation
+              <span class="material-icon" aria-hidden="true">launch</span>
+            </a>
+          </li>
```

Comment:
Turns out we don't need any of the `tabindex` values on nav-toggle button or the links in the dropdown list (something to update in the Style Guide as well).